base/common/python/pki/nssdb.py

```diff
@@ -1196,7 +1252,14 @@ def get_trust(self, nickname, token=None):
                 logger.warning('certutil returned non-zero exit code (bug #1539996)')
 
             re_compile = re.compile(r'^' + fullname + r'\s+(\S+)$', re.MULTILINE)
-            cert_trust = re.search(re_compile, output.decode()).group(1)
+            re_line = re.search(re_compile, output.decode())
+            if re_line != None:
+                print(re_line)
+                cert_trust = re.search(re_compile, output.decode()).group(1)
+                logger.debug('cert_trust : %s', cert_trust)
+            else:
+                logger.debug('re_line for re.search is None')
+                print("output.decode : %s", output.decode())
```

Comment:
Are these changes necessary? It seems to be executing the same regular expression twice.